scripts/add_webhook.sh

```diff
@@ -16,19 +16,24 @@ if [ -z "${GITHUB_SECRET_TOKEN}" ]; then
   exit 1
 fi
 
+if [ -z "${GITHUB_AUTH_TOKEN}" ]; then
```

Comment:
Should this be GITHUB_OAUTH_TOKEN ? Since that is already an environment variable per https://github.com/theforeman/prprocessor#environment-variables

Reply:
@ehelms right, I generated an OAuth token on my personal account (since I have owner I can touch everything). I don't think PrProcessor itself is meant to be able to alter the webhooks, but maybe I'm wrong on that.